srcpkgs/bashmount/template

```diff
@@ -1,16 +1,16 @@
 # Template file for 'bashmount'
 pkgname=bashmount
 version=3.2.0
-revision=1
+revision=2
 conf_files="/etc/$pkgname.conf"
 depends="bash util-linux"
 short_desc="Tool to mount and unmount removable media"
 maintainer="Daniel Lewan <vision360.daniel@gmail.com>"
-noarch=yes
-license="GPL-2"
-homepage="https://github.com/jamielinux/$pkgname"
-distfiles="$homepage/archive/$version.tar.gz"
+license="GPL-2.0-or-later"
+homepage="https://github.com/jamielinux/bashmount"
+distfiles="https://github.com/jamielinux/bashmount/archive/$version.tar.gz"
 checksum=32f328e1921f60b429e2ab389ecc314f5bd7545ac4385c7c7918d08ebe9ecc22
+archs=noarch
```

Comment:
should be where noarch=yes generally is

Reply:
I get it, seems like xlint was not updated yet, I'll change that